test/functional/page_objects/visualize_page.js

```diff
@@ -776,6 +777,11 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
       .getAttribute('height');
     }
 
+    async getDataTableHeaders() {
+      const dataTableHeader = await retry.try(async () => await find.byCssSelector('table.table.table-condensed thead'));
+      return await dataTableHeader.getVisibleText();
```

Comment:
Can we please not add more CSS selectors to the functional tests and instead add `data-test-subj` attributes to selection targets and use the `testSubjects` service?